### PR TITLE
#167739996 updating internal pages

### PR DESCRIPTION
**Pull request by Niyonsengaeric**
#### What does this PR do?
**Updating  Internal Pages**
#### Description of Task to be completed?
**Internal Pages implementation**
- I have Fixed the Header and Footer on BOOKINGS, TRIPS, DASHBOARD and  CONTACT US pages
- I have Implemented the filter for list of data on  bookings , Trips and on the dashboard pages
- I have implement the table colors for good Display on the Bookings and trips Pages
- I have resized Input text box , on BOOKING TICKET,Sign Up, Contact us and Login pages
- I have removed Unused space and Unnecessary comments

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/167739996
#### Screenshots (if appropriate)

1 header
1 login body
1 trips body
1 book ticket body
1 bookings body
1 dashbord body
1 footer

#### Questions:
N/A

### CODE REVIEW

File: UI/css/aboutus.css [hunk 1]
```diff
@@ -18,17 +21,10 @@ body{height: auto;}
...
-.about-details h1{
+.about-details h1 {
   color: #000;
   font-size: 25;
   font-weight: bold;
-  
- font-family: 
-
-
 }
```
Comment (oesukam) [inline UI/css/aboutus.css]: Blank line required

File: UI/css/book.css [hunk 2]
```diff
@@ -61,28 +65,34 @@
...
+
+.line-on-side span {
   background-color: #fff;
 }
 
 #prices input {
   text-align: center;
   width: 50%;
-}
+}
```
Comment (oesukam) [inline UI/css/book.css]: Blank line required

File: UI/css/bookings.css [hunk 3]
```diff
@@ -64,9 +92,19 @@ div a {text-decoration: none;}
...
 
-.container h4{text-align: center;}
-tr a {text-decoration: none;color: black;}
-tr a:hover {text-decoration: none;color: #23b4ca;;}
\ No newline at end of file
+tr a:hover {
+  text-decoration: none;
+  color: #23b4ca;
+  ;
+}
```
Comment (oesukam) [inline UI/css/bookings.css]: Blank line required

File: UI/css/contact.css [hunk 4]
```diff
@@ -1,90 +1,92 @@
...
-  }
\ No newline at end of file
+.contact-page .form h2 {
+  color: #656565;
+  font-size: 15px;
+  line-height: 25px;
+  font-weight: 100;
+  border-bottom: 1px dashed #d6d6d6;
+  text-align: center;
+}
```
Comment (oesukam) [inline UI/css/contact.css]: Blank line required

File: UI/css/dashboard.css [hunk 5]
```diff
@@ -70,8 +71,19 @@
...
+
+tr a {
+  text-decoration: none;
+  color: black;
+}
+
+tr a:hover {
+  text-decoration: none;
+  color: #23b4ca;
+}
```
Comment (oesukam) [inline UI/css/dashboard.css]: Blank line required

File: UI/css/login.css [hunk 6]
```diff
@@ -1,59 +1,64 @@
...
+  font-size: 18px;
+  text-decoration: none;
 }
 
-form h2{text-align: center;text-transform: uppercase;}
\ No newline at end of file
+form h2 {
+  text-align: center;
+  text-transform: uppercase;
+}
```
Comment (oesukam) [inline UI/css/login.css]: Blank line required

File: UI/css/trips.css [hunk 7]
```diff
@@ -64,9 +85,19 @@ div a {text-decoration: none;}
...
 }
-.container h4{text-align: center;}
-tr a {text-decoration: none;color: black;}
-tr a:hover {text-decoration: none;color: #23b4ca;;}
 
+tr a:hover {
+  text-decoration: none;
+  color: #23b4ca;
+  ;
+}
```
Comment (oesukam) [inline UI/css/trips.css]: Blank line required

File: UI/pages/aboutus.html [hunk 8]
```diff
@@ -67,13 +89,12 @@ <h2>Mission</h2>
...
-
   <div class="icon">
     <a href="#"><img src="../images/facebook.png"></a>
     <a href="#"><img src="../images/twitter.png"></a>
     <a href="#"><img src="../images/youtube.png"></a>
   </div>
+  <p>© copyright 2019 <a href="aboutus.html"> Way Farer.</a> All rights Reserve</p>
 </div>
 
 </html>
```
Comment (oesukam) [inline UI/pages/aboutus.html]: Blank line required

File: UI/pages/signup.html [hunk 9]
```diff
@@ -134,29 +155,21 @@ <h1 style="text-align: center;">New User</h1>
...
       <a href="#"><img src="../images/facebook.png"/></a>
       <a href="#"><img src="../images/twitter.png"/></a>
       <a href="#"><img src="../images/youtube.png"/></a>
     </div>
+    <p>
+      © copyright 2019 <a href="aboutus.html"> Way Farer.</a> All rights Reserve
+    </p>
   </div>
-</html>
+</html>
```
Comment (oesukam) [inline UI/pages/signup.html]: Blank line required

File: UI/pages/spec.html [hunk 10]
```diff
@@ -49,12 +69,13 @@ <h1 style="text-align: center;">Ticket Details</h1>
...
+
 
   <div class="icon">
     <a href="#"><img src="../images/facebook.png"></a>
     <a href="#"><img src="../images/twitter.png"></a>
     <a href="#"><img src="../images/youtube.png"></a>
   </div>
+  <p>© copyright 2019 <a href="aboutus.html"> Way Farer.</a> All rights Reserve</p>
 </div>
 </html>
```
Comment (oesukam) [inline UI/pages/spec.html]: Blank line required

File: UI/pages/spectrip.html [hunk 11]
```diff
@@ -47,13 +71,11 @@ <h1 style="text-align: center;">Trip-details</h1>
...
-
   <div class="icon">
     <a href="#"><img src="../images/facebook.png"></a>
     <a href="#"><img src="../images/twitter.png"></a>
     <a href="#"><img src="../images/youtube.png"></a>
   </div>
+  <p>© copyright 2019 <a href="aboutus.html"> Way Farer.</a> All rights Reserve</p>
 </div>
-
 </html>
```
Comment (oesukam) [inline UI/pages/spectrip.html]: Blank line required

**Review by oesukam — CHANGES_REQUESTED** [review]

File: UI/css/aboutus.css (hunk 1 above)
Comment (Niyonsengaeric) [reply UI/css/aboutus.css]: i have solve it

File: UI/css/book.css (hunk 2 above)
Comment (Niyonsengaeric) [reply UI/css/book.css]: i have add it!

File: UI/css/bookings.css (hunk 3 above)
Comment (Niyonsengaeric) [reply UI/css/bookings.css]: i have add it!

File: UI/css/contact.css (hunk 4 above)
Comment (Niyonsengaeric) [reply UI/css/contact.css]: i have add it!

File: UI/css/dashboard.css (hunk 5 above)
Comment (Niyonsengaeric) [reply UI/css/dashboard.css]: i have add it!

File: UI/css/trips.css (hunk 7 above)
Comment (Niyonsengaeric) [reply UI/css/trips.css]: i have add it!

File: UI/pages/signup.html (hunk 9 above)
Comment (Niyonsengaeric) [reply UI/pages/signup.html]: Blank line required

**Review by oesukam — APPROVED** [review]

**Pull request merged**